src/gamestates/TestState.cpp

```diff
@@ -2,12 +2,12 @@
 
 #include "TestState.h"
 
-#include "GameObjectMediator.h"
-#include "Universe.h"
-#include "CollisionDetector.h"
-#include "BasicEnemy.h"
-#include "GraphicsHandler.h"
-#include "MainCharacter.h"
+#include "src/GameObjectMediator.h"
```

Comment:
We can run the build within the src directory too, and the make all script will just cd into the src before running it's current stuff.